### DISCUSSION TITLE
可以简单理解为我们这次共学的产物会是一串 web3:// 开头的链接对吗？

### DISCUSSION THREAD

**Discussion by ztm0929** [IntensiveCoLearning/Web3-URL#52] [Q&A] [answered]
本人纯小白，简单看了下文档，得出这个想法🤣

**Comment by brucexu-eth** [comment, marked as answer]
应该不是，应该是基于 Web3 URL 实现的应用吧？比如一个博客或者论坛，存储在链上，可以通过 Web3:// 直接访问什么的？我猜的，我也在学习

**Reply by ztm0929** [reply]
对，我看了老师的PPT（抱歉英文太烂，没完全理解hhh）

他有提到现在的链上内容已经有了（博客、网盘等等），但是人们依然需要https协议（即DNS解析服务）才能够访问，因此希望推广web3协议，我目前理解来看，重点可能优先是**将 https:// 转为 web3://** （即转换已有的链上内容访问方式）

当然我有这个观点也主要是因为我不会上链哈哈哈，可能其他有经验的同学自行创建会更快更适合些。

**Reply by brucexu-eth** [reply]
有任何不理解的都可以随时创建 discussion！正好给他们一些反馈，方便优化内容

**Comment by luffythink** [comment]
我觉得可以简单理解为：共学的产物会是一串 web3:// 开头的链接，且这串链接能正确使用 🤔，比如通过 web3://xxx 链接，实现直接访问链上的什么内容，这些内容是如何呈现的，这些需基于 web3:// 如何去开发实现。
